libs/subliminal_patch/providers/ktuvit.py

```diff
@@ -0,0 +1,440 @@
+                sanitize(self.title)
+                in (
+                    sanitize(name) for name in [video.series] + video.alternative_series
+                )
+            ):
+                matches.add("series")
+            # season
+            if video.season and self.season == video.season:
+                matches.add("season")
+            # episode
+            if video.episode and self.episode == video.episode:
+                matches.add("episode")
+            # imdb_id
+            if video.series_imdb_id and self.imdb_id == video.series_imdb_id:
+                matches.add("series_imdb_id")
+            # guess
+            matches |= guess_matches(video, guessit(self.release, {"type": "episode"}))
+        # movie
+        elif isinstance(video, Movie):
+            # guess
+            matches |= guess_matches(video, guessit(self.release, {"type": "movie"}))
+
+            # title
+            if video.title and (
+                sanitize(self.title)
+                in (sanitize(name) for name in [video.title] + video.alternative_titles)
+            ):
+                matches.add("title")
+
+        return matches
+
+
+class KtuvitProvider(Provider):
+    """Ktuvit Provider."""
+
+    languages = {Language(l) for l in ["heb"]}
+    server_url = "https://www.ktuvit.me/"
+    sign_in_url = "Services/MembershipService.svc/Login"
+    search_url = "Services/ContentProvider.svc/SearchPage_search"
+    movie_info_url = "MovieInfo.aspx?ID="
+    episode_info_url = "Services/GetModuleAjax.ashx?"
+    request_download_id_url = "Services/ContentProvider.svc/RequestSubtitleDownload"
+    download_link = "Services/DownloadFile.ashx?DownloadIdentifier="
+    subtitle_class = KtuvitSubtitle
+
+    _tmdb_api_key = "a51ee051bcd762543373903de296e0a3"
+
+    def __init__(self, email=None, hashed_password=None):
+        if any((email, hashed_password)) and not all((email, hashed_password)):
+            raise ConfigurationError("Email and Hashed Password must be specified")
+
+        self.email = email
+        self.hashed_password = hashed_password
+        self.logged_in = False
+        self.session = None
+        self.loginCookie = None
+        self.headers = None
+
+    def initialize(self):
+        self.session = Session()
+        self.session.headers["User-Agent"] = "Subliminal/{}".format(__short_version__)
+
+        # login
+        if self.email and self.hashed_password:
+            logger.info("Logging in")
+
+            data = {"request": {"Email": self.email, "Password": self.hashed_password}}
+            headers = {
+                'Accept-Encoding': 'gzip',
+                'Accept-Language': 'en-us,en;q=0.5',
+                'Pragma': 'no-cache',
+                'Cache-Control': 'no-cache',
+                'Content-Type': 'application/json',
+                'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/78.0.3904.97 Safari/537.36',
```

Comment:
Why are you overwriting user-agent here?

Reply:
👍🏻